ansible_collections/arista/avd/roles/eos_designs/templates/l3_edge/router-bgp-neighbors.j2

```diff
@@ -4,7 +4,7 @@ router_bgp:
 {%     for bgp_neighbor_key in l3_edge_data.bgp_neighbors | arista.avd.natural_sort %}
 {%         set bgp_neighbor = l3_edge_data.bgp_neighbors[bgp_neighbor_key] %}
     {{ bgp_neighbor_key }}:
-      remote_as: {{ bgp_neighbor.remote_as }}
+      remote_as: "{{ bgp_neighbor.remote_as }}"
```

Comment:
you should add quotes around the `local_as` in line 14.

Reply:
Quotes added.